src/main/java/subway/line/dao/LineDao.java

```diff
@@ -53,13 +55,13 @@ public Line findById(Long id) {
     }
 
     public void update(Line newLine) {
-        String sql = "update LINE set name = ?, color = ? where id = ?";
-        jdbcTemplate.update(sql, new Object[]{newLine.getName(), newLine.getColor(), newLine.getId()});
+        String sql = "update LINE set name = ?, color = ?, extra_fare = ? where id = ?";
+        jdbcTemplate.update(sql, new Object[]{newLine.getName(), newLine.getColor(),newLine.getFare(), newLine.getId()});
```

Comment:
`jdbcTemplate.update(String sql, @Nullable Object... args)` 형태라서,
new Object[] 없이 가변 인자 형태로 호출 가능합니다 😄 
```suggestion
        jdbcTemplate.update(sql, newLine.getName(), newLine.getColor(),newLine.getFare(), newLine.getId());
```